ads/taboola.js

```diff
@@ -38,7 +38,7 @@ export function taboola(global, data) {
 
   // copy none blacklisted attribute to the 'params' map
   Object.keys(data).forEach(k => {
-    if (blackList.indexOf(k) === -1) {
+    if (!blackList.includes(k)) {
```

Comment:
So everything in this file cannot be updated, because the ads directory is run in foreign iframes. This means they will not have the Array#includes polyfill installed in them.